app/javascript/src/components/menus/activated_menu_activator.js

```diff
@@ -64,10 +68,12 @@ class ActivatedMenuActivator {
   #createNode() {
       const $node = $(
         createElement("button", 
-                      this.#config.activator_text, 
+                      undefined, 
                       this.#config.activator_classname
         )
       );
```

Comment:
I see what you're going for here. but it's not good to put this much HTML in the JS component code, especially when it's aiming to resolve a design-only issues (the icon) or hardcoding GDS (so visual only) classnames. 

Also, as per the notes on Configuration above, you can pass in an activator element to use, so all this could be in the template. When nothing is specified the component uses the above createElement() function to inject a basic <button>